packages/selinux/collection.yaml

```diff
@@ -5,13 +5,13 @@ packages:
     labels:
       github.repo: "k3s-selinux"
       github.owner: "k3s-io"
-      github.tag: "0.3.testing.0"
```

Comment:
did you gave it a shot locally? I'm pretty sure the issue is also that it would try to bump to an invalid version